controllers/apps.js

```diff
@@ -63,6 +63,40 @@ module.exports = {
       .catch(error => { console.log(error); });
   },
 
+  async update(req, res) {
+    if (!req.file || req.file.mimetype !== 'application/zip') {
+      return res.render('apps/update', { errors: [{ message: 'Please attach a .zip file of your application.' }] })
+    }
+
+    const app = {
+      title: req.body.title,
+      path: `uploads/${req.body.title}`,
+      filename: req.file.filename + '.zip',
+      network: `${req.body.title}_default`
+    };
+
+    await moveApplicationFile(req);
+
+    App
+       .findByPk(req.params.appId)
+       .then((app) => app.update({ filename: app.filename }))
+       .then((app) => {
```

Comment:
Looks like these lines are indented with 4 spaces instead of 2

Reply:
That's because it's following the format:

```javascript
module.exports = {
  async update() {
    App
      .then()
      .then()
      .catch()
  }
}
```